### PR TITLE
feat: added payout methods section

### LINKED ISSUE #873: feature: Add a section where user can add their payout method
A place where user can come and add their
1. upi
2. solana address

### PR DESCRIPTION
**Pull request by ajeeshRS**
### PR Fixes:
-  Added payout methods section

### Features :
- User can add their upi and solana addresses
- Validation of upi and solana address format using regex
- Success and error messages
- Only 2 upi and 2 solana addresses can be added for a user
- Avoided duplicate values
- User can delete a particular address
- Cross device compatability

Resolves #873

[Take-your-development-skills-from-0-to-100-and-join-the-100xdevs-community.webm](https://github.com/user-attachments/assets/e90d2b44-dcb5-4fea-8a43-b6f78453b3dd)

### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] I assure there is no similar/duplicate pull request regarding same issue

### CODE REVIEW

**Comment by ajeeshRS** [comment]
@hkirat Could you please review the PR when you have a moment? Thanks!

**Comment by siinghd** [comment]
@ajeeshRS fix yarn lock

**Comment by siinghd** [comment]
@ajeeshRS why only two upi/solana wallets?

**Comment by ajeeshRS** [comment]
@siinghd just to make it simple and manageable for now

**Comment by ajeeshRS** [comment]
@siinghd sounds good,thanks!

File: src/actions/payoutMethods/schema.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    .refine((value) => /^[A-Za-z0-9]{44}$/.test(value), {
+      message: 'Enter a valid Solana address',
+    })
+    .optional(),
+});
+
+export const upiIdInsertSchema = z.object({
+  upiId: z
+    .string()
+    .refine((value) => /^[0-9A_Za-z._-]{2,256}@[A_Za-z]{2,64}$/.test(value), {
```
Comment (divu777) [inline src/actions/payoutMethods/schema.ts:21]: Just little minor issue with the regix , i think its supposed to be [0-9A-Za-z._-]{2,256}@[A-Za-z]

**Review by divu777 — COMMENTED** [review]
i just had this issue while setting up the cms

File: src/actions/payoutMethods/schema.ts (hunk 1 above)
Comment (ajeeshRS) [reply src/actions/payoutMethods/schema.ts:21]: Thanks for catching that! I used A_Z instead of A-Z by mistake.I'll fix the regex to get it right.

**Pull request merged**